packages/create-llama/index.ts

```diff
@@ -153,7 +153,7 @@ const program = new Commander.Command(packageJson.name)
     "--tools <tools>",
     `
 
-  Specify the tools you want to use by providing a comma-separated list. For example, 'google_search,wikipedia'. Use 'none' to not using any tools.
+  Specify the tools you want to use by providing a comma-separated list. For example, 'wikipedia.WikipediaToolSpec,google.GoogleSearchToolSpec'. Use 'none' to not using any tools.
```

Comment:
this is considered as a breaking change when we update from previous version (google_search -> google.GoogleSearchToolSpec, wikipedia -> wikipedia.WikipediaToolSpec).

We can create a shortcut name for it to keep the same tool name as it is but use the too_package.TooSpecClassName convention is more predictable.